lib/master.js

```diff
@@ -134,18 +135,33 @@ function createLookup( workers ){
   // Once all responses for search `id` have been pooled, assemble the
   // administrative names object and pass it to that search's callback.
   function completeSearch( id ){
-    var responses = responseMap[ id ];
+    var resps = responseMap[ id ];
     delete responseMap[ id ];
 
     var result = {};
-    result.alpha3 = responses.admin1.qs_adm0_a3;
-    result.admin0 = responses.admin1.qs_adm0;
-    result.admin1 = responses.admin1.qs_a1;
-    result.admin2 = responses.admin2.qs_a2 || responses.neighborhood.name_adm2;
-    result.local_admin = responses.local_admin.qs_la;
-    result.locality = responses.locality.qs_loc;
-    result.neighborhood = responses.neighborhood.name;
-    responses.cb( result );
+    result.alpha3 = resps.admin1.qs_adm0_a3;
+    result.admin0 = resps.admin1.qs_adm0;
+    result.admin1 = resps.admin1.qs_a1;
+    result.admin2 = resps.admin2.qs_a2 || resps.neighborhood.name_adm2;
+    result.local_admin = resps.local_admin.qs_la;
+    result.locality = resps.locality.qs_loc;
+    result.neighborhood = resps.neighborhood.name;
+    result.population = resps.local_admin.qs_pop || resps.locality.qs_pop;
+
+    var popularities = [
+      resps.admin1.popularity,
+      resps.admin2.popularity,
+      resps.local_admin.popularity,
+      resps.locality.popularity,
+      resps.neighborhood.popularity
+    ];
+    result.popularity = popularities.reduce( function ( acc, curr ){
+      return ( typeof curr === 'number' ) ?
+        acc + curr :
+        acc;
+    });
+    console.log( result );
```

Comment:
is this for debugging purposes? should we delete this line before merging into master?